.github/actions/build/action.yaml

```diff
@@ -46,7 +46,7 @@ runs:
       with:
         repository: picoreplayer/lms-update-script
         path: platforms/pcp
-
+        ref: prebuilt_tcz
```

Comment:
Oh... we don't really need this, do we? The init file should not have changed?

Reply:
The init script is the same in both branches.  This is to keep the update script separate, until I do a planned migration from the Custom to prebuilt.